cudamapper/src/matcher.cu

```diff
@@ -326,8 +329,9 @@ Matcher::Matcher(const Index& index, uint32_t query_target_division_idx)
         read_id_to_pointer_arrays_section_h.clear();
         read_id_to_pointer_arrays_section_h.shrink_to_fit();
 
-        CGA_LOG_INFO("Allocating {} bytes for anchors_d", total_anchors * sizeof(Anchor));
-        device_buffer<Anchor> anchors_d(total_anchors);
+        auto num_anchors_so_far = anchors_d_.size();
+        anchors_d_.resize(num_anchors_so_far + total_anchors);
+        Anchor* anchors_d = thrust::raw_pointer_cast(anchors_d_.data()) + num_anchors_so_far;
```

Comment:
Why has this been marked as resolved? the code is unchanged?

Reply:
From slack

>@Milos Maric - Thanks for your review of https://github.com/clara-genomics/ClaraGenomicsAnalysis/pull/214. I don't have time to address them all this week, will work on the P0 ones that could be a blocker.
- As far your comment on using anchors_d_.data().get() instead of thrust::raw_pointer_cast(anchors_d_.data())  - you can't use *.get() on the host_vector and to be consistent, the recommended way is to use thrust::raw_pointer_cast.

Reply:
> Why has this been marked as resolved? the code is unchanged?

I seem to have pushed the file without the suggested change, I have corrected it now.